gui/network/migrations/0006_globalconfiguration_gc_domains.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-23 11:07
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('network', '0005_vlan_pcp.py')
```

Comment:
Yes I wrote the migration script manually and accidently added .py in the end. 
I will remove it